docsource/modules150-160.rst

```diff
@@ -138,7 +138,7 @@ Module coverage 15.0 -> 16.0
 +-------------------------------------------------+----------------------+-------------------------------------------------+
 | event_sms                                       |                      |No DB layout changes.                            |
 +-------------------------------------------------+----------------------+-------------------------------------------------+
-| |del| fetchmail                                 |                      |                                                 |
+| |del| fetchmail                                 | Nothing to do        | Merged into mail.                               |
```

Comment:
```suggestion
| |del| fetchmail                                 | Done        | Merged into mail.                               |
```

(as something is done in the apriori.py file).

Reply:
thanks for the review @legalsylvain! fixed and updated git comment.